### ISSUE TITLE
add count support for insert mode commands

### ISSUE THREAD

**Issue by alexherbo2** [mawww/kakoune#1324]
to insert only the nth selection (relative to the main)

**Comment by lenormf** [comment]
Dupe of #1106.

**Comment by Delapouite** [comment]
Yes, this issue can be closed:  https://github.com/mawww/kakoune/issues/1106#issuecomment-270183852
> Another reason was to keep the count parameter for an eventual other use, like inserting only for a given selection.

**Comment by danr** [comment]
Inserting with just the main cursor is 

```
    Z<space>itext<esc>z
```

(use `'` and `<a-'>` to change main cursor)

**Comment by alexherbo2** [comment]
@danr I still think there should be a built-in and easy way going to access this functionality.

**Comment by alexherbo2** [comment]
https://github.com/occivink/kakoune-interactive-itersel
